src/components/HighlightedText/index.js

```diff
@@ -1,32 +1,31 @@
 import React from 'react';
-import PropTypes from 'prop-types';
+import PropTypes, { bool } from 'prop-types';
```

Comment:
we normally do PropTypes.bool in order to keep consistency let do it in that way